client/main.lua

```diff
@@ -108,15 +108,15 @@ local function CreateZone(type, garage, index)
                 if garage.type == "house" then
                     text = Lang:t("info.park_e")
                 else
-                    text = Lang:t("info.park_e") .. "<br>" .. garage.label
+                    text = Lang:t("info.park_e") .. "  \n" .. garage.label
```

Comment:
```suggestion
                    text = Lang:t("info.park_e") .. "\n" .. garage.label
```

Why the spaces?

Reply:
You have to do spaces?

"\n" does NOT work

Reply:
You need double spaces for it to work